Mathlib/LinearAlgebra/RootSystem/Finite/CanonicalBilinear.lean

```diff
@@ -132,6 +133,71 @@ lemma rootForm_root_self (j : ι) :
     P.RootForm (P.root j) (P.root j) = ∑ (i : ι), (P.pairing j i) * (P.pairing j i) := by
   simp [rootForm_apply_apply]
 
+theorem range_polarization_domRestrict_le_span_coroot :
+    LinearMap.range (P.Polarization.domRestrict (Submodule.span R (Set.range P.root))) ≤
```

Comment:
How about we add:
```lean
open Set
open Submodule (span)
```
at the top of the file and then we can write this as:
```suggestion
    LinearMap.range (P.Polarization.domRestrict (span R (range P.root))) ≤
```

Or maybe even introduce:
```lean
abbrev rootSpan := span R (range P.root)
abbrev corootSpan := span R (range P.coroot)
```
over in `RootSystem.Defs`?